frontend/src/modules/dashboard/components/ValueInfo.tsx

```diff
@@ -294,32 +348,39 @@ const ControlValuesDisplay = ({
 /**
  * ValueInfo
  *
- * @component Component for showing information.
+ * @component Component for showing information based on layout configured.
  *
- * Uses the [[Props]] interface
+ * Uses the [[ValueInfoProps]] interface
  *
  * @returns {JSX.Element}
  *
  */
+// TODO: we should delete this component if it's not being used in any current code;
+// its structure is weird and its proptypes doesn't pass the linter
```

Comment:
Is this TODO comment resolved now? It looks like you made `ValueInfo` have a reasonable structure and reasonable proptypes now.

Reply:
Yes this is resolved now.